js/main.js

```diff
@@ -932,7 +933,14 @@ var mapView = {
           if (a.creation_time < b.creation_time) return 1;
           return 0;
         });
-        break;
+        break;      
+      case 'level':
```

Comment:
In my opinion it should be creation_time, but nevertheless it will work that way too because the data attribute of the button `data-sort="time"` and the case `case 'time':` match. In your code the data attribute is `data-sort="lvl"` and the case is `case 'level'`. They do not match so the case won't be selected and the default sorting (by cp) will be used. Did you test that code locally?